cclibs/scripts/scripts.js

```diff
@@ -13,23 +13,24 @@
 import { setLibs } from './utils.js';
 
 // Add project-wide style path here.
-const STYLES = '';
+const STYLES = '/cclibs/styles/styles.css';
 
 // Use '/libs' if your live site maps '/libs' to milo's origin.
 const LIBS = 'https://milo.adobe.com/libs';
 
 // Add any config options.
 const CONFIG = {
-  // codeRoot: '',
-  // contentRoot: '',
-  // imsClientId: 'college',
-  // geoRouting: 'off',
-  // fallbackRouting: 'off',
+  codeRoot: '/cclibs',
+  contentRoot: '/creativecloud',
+  imsClientId: 'ccmilo',
   locales: {
     '': { ietf: 'en-US', tk: 'hah7vzn.css' },
     de: { ietf: 'de-DE', tk: 'hah7vzn.css' },
     kr: { ietf: 'ko-KR', tk: 'zfo3ouc' },
+    TH_th: { ietf: 'TH–th', tk: 'zfo3ouc' },
```

Comment:
This doesn't seem to follow the correct pattern. To my knowledge, we do not use uppercase TH_ on adobe.com.

Reply:
Changed teh locale as per existing format